api.go

```diff
@@ -378,7 +378,8 @@ func CreateContainer(podID string, containerConfig ContainerConfig) (*Pod, *Cont
 	}
 
 	// Store it.
-	err = c.storeContainer()
+	cImpl := c.(*Container)
```

Comment:
Not needed if we agree that `createContainer()` should not return `VCContainer` type.